### ISSUE TITLE
请问有论文中最佳效果的预训练好的模型吗

### ISSUE THREAD

**Issue by irsLu** [easonyang1996/CS-CO#2]

**Comment by easonyang1996** [comment]
Of course, you can download it from https://drive.google.com/file/d/1h7SSzndEm486-gAbwtKA81hO4TA84Ita/view?usp=sharing

**Comment by irsLu** [comment]
> Of course, you can download it from https://drive.google.com/file/d/1h7SSzndEm486-gAbwtKA81hO4TA84Ita/view?usp=sharing

非常感谢~，再次打扰您了，请问还有一个问题是我通过您的代码csco_vahadnae.py来实现H、E分离的图片结果与您的效果有点不一致，我代码是否存在错误呢？
代码如下：

v = vahadane(fast_mode=0)
I0 = imageio.imread('./2.png')
W,H = v.stain_separate(I0)

_H,_W,_C = I0.shape
Ih = (255 * np.resize(np.exp(-H[0, :]), (_H, _W, 1)) ).astype(np.uint8)
Ie = (255 * np.resize(np.exp(-H[1, :]), (_H, _W, 1)) ).astype(np.uint8)

imageio.imsave('./h.png', Ih)
imageio.imsave('./e.png', Ie)

**Comment by jjl001** [comment]
@easonyang1996  
请问一下这个是cs的模型吗？ https://drive.google.com/file/d/1h7SSzndEm486-gAbwtKA81hO4TA84Ita/view?usp=sharing
有没有cs-co的模型？

**Comment by HHHedo** [comment]
> Of course, you can download it from https://drive.google.com/file/d/1h7SSzndEm486-gAbwtKA81hO4TA84Ita/view?usp=sharing

Thanks for your great work! 
I wonder whether you can release other pre-trained models (e.g., the models of chen_MIA and XIE_miccai), which can help the following works a lot.
